vendor/github.com/knative/test-infra/scripts/e2e-tests.sh

```diff
@@ -40,7 +40,7 @@ readonly E2E_BASE_NAME=k$(basename ${REPO_ROOT_DIR})
 readonly E2E_CLUSTER_NAME=$(build_resource_name e2e-cls)
 readonly E2E_NETWORK_NAME=$(build_resource_name e2e-net)
 readonly E2E_CLUSTER_REGION=us-central1
-readonly E2E_CLUSTER_ZONE=${E2E_CLUSTER_REGION}-a
+# readonly E2E_CLUSTER_ZONE=${E2E_CLUSTER_REGION}-a
```

Comment:
why this change? If we don't need it -- just remove?

Reply:
Too late, this is just vendoring test-infra :(

Reply:
this has been removed in latest test-infra repo - https://github.com/knative/test-infra/commit/9183d311c48ac8bf5ebc62786bbb1167a54d3640. It will be picked up by the next update.